src/service.js

```diff
@@ -119,8 +126,8 @@ function authLocalMgntMethods (options) {
             return await sendResetPwd(
               options,
               data.value,
-              data.notifierOptions,
-              passwordField
+              passwordField,
+              data.notifierOptions
```

Comment:
Is there a specific reason for changing the order of these

Reply:
I wanted to be consistent on how I passed in `data.notifierOptions`